site/docs/advanced/deployment.md

```diff
@@ -18,8 +18,7 @@ Here is a list of things that you may want to keep in mind when hosting a large
 
 1. Send files by path or `Buffer` instead of by stream, or at least make sure you [know the pitfalls](./transformers.md#use-cases-of-transformer-functions).
 2. Use `bot.on("callback_query:data")` as the fallback handler to [react to all callback queries](../plugins/keyboard.md#responding-to-clicks).
```

Comment:
Should we do the same for inline queries, commands, chosen inline results, and game queries?

Reply:
In a separate PQ ofc